Docs: start basic template (#1666)
microsoft/VFSForGit: docs/index.md

> VFS for Git: Virtualized File System for Git

Reviewer: (not sure where to place this, so i randomly picked this line.)

Should we have a footnote about dropping support for the mac for external reasons.  in case a reader had already seen the earlier version and was wondering where it went (and why) ?

Author: That's a good thing for the FAQ, probably. Let me make a note in the issue to add that (along with a description of how VFS for Git and Scalar fit together in our strategy).

============================================

VFS stands for Virtual File System. VFS for Git virtualizes the file system

> beneath your Git repo so that Git and all tools see what appears to be a

Reviewer 2: ```suggestion
beneath your Git worktree so that Git and all tools see what appears to be a

```

> normal repo, but VFS for Git only downloads objects as they are needed.

Reviewer 2: ```suggestion
regular worktree, but VFS for Git only downloads objects as they are needed.

```

Author: "regular working directory" works here.

> VFS for Git also manages the files that git will consider, to ensure that

Reviewer 2: ```suggestion
VFS for Git also manages the files that Git will consider, to ensure that

```

Author: Thanks! This is leftover from the readme, but I tried to check the rest of the docs, too.

Git operations such as `status`, `checkout`, etc., can be as quick as
possible because they will only consider the files that the user has
accessed, not all files in the repo.

> Installing on Windows

Reviewer: If we're dropping all references to the Mac version, we can simplify this section heading.
